src/v/raft/consensus.cc

```diff
@@ -2011,4 +2045,37 @@ void consensus::suppress_heartbeats(
     }
 }
 
+voter_priority consensus::next_target_priority() {
+    return voter_priority(
+      std::max<int32_t>(_target_priority * target_priority_update_factor, 1));
```

Comment:
so this is weird. you do max int32_t but you use `uint` for the type? wrong type?

Reply:
good point, a typo